<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVDisparityTuner</class>
 <widget class="QMainWindow" name="OpenCVDisparityTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1859</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCVDisparityTuner 1.0 by Julien Jomier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="loadLeftImageButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load left image...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="processButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>620</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadRightImageButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load right image...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Num Disparities</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="numberOfDisparities">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>140</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>SADWindowSize</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="SADWindowSize">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="minDisparity">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="value">
          <number>-4</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>minDisparity</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>240</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>preFilterCap</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="preFilterCap">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>290</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>uniquenessRatio</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="uniquenessRatio">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="disp12MaxDiff">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>360</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>340</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>disp12MaxDiff</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="fullDP">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>390</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Full DP</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="speckleWindowSize">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>430</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>410</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>speckleWindowSize</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>460</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>speckleRange</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="speckleRange">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>480</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>510</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>P1</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="p1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>530</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>600</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="p2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>580</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>2400</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>560</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>P2</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CQtOpenCVViewerGl" name="imageViewerLeft" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="CQtOpenCVViewerGl" name="imageViewerRight" native="true"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="CQtOpenCVViewerGl" name="imageViewerDisparity" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtOpenCVViewerGl</class>
   <extends>QWidget</extends>
   <header>cqtopencvviewergl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
